pdns/recursordist/rec-main.cc

```diff
@@ -3045,16 +3047,19 @@ static RecursorControlChannel::Answer* doReloadLuaScript()
     }
     catch (std::runtime_error& ex) {
       string msg = std::to_string(RecThreadInfo::id()) + " Retaining current script, could not read '" + fname + "': " + ex.what();
-      g_log << Logger::Error << msg << endl;
+      SLOG(g_log << Logger::Error << msg << endl,
+           log->error(Logr::Error, ex.what(), "Retaining current script, could not read new script"));
```

Comment:
Aren't `RecThreadInfo::id()` and `fname` missing here and in the two calls below?

Reply:
The `log` object already includes `fname`, see line 3035. `tid` is added by the structured logging backend on all calls.